demosplan/DemosPlanCoreBundle/Logic/Statement/StatementService.php

```diff
@@ -2032,23 +1950,11 @@ public function addVote($statementId, User $user)
             ];
 
             // only one vote per user per statement
-            $vote = $this->statementVoteRepository->findOneBy([
+            return $this->statementVoteRepository->findOneBy([
                 'user'      => $user->getId(),
                 'statement' => $statementId,
                 'deleted'   => false,
                 'active'    => true, ]);
-
-            // user already voted this statement?
-            if (null === $vote) {
-                $vote = $this->statementVoteRepository->add($data);
```

Comment:
`$this->statementVoteRepository->add($data);` really not needed anymore?

Reply:
Good catch, thx! f9c4a333